livingatlas/debian/rules

```diff
@@ -24,7 +24,7 @@ ifeq ($(filter nobuildjar,$(DEB_BUILD_PROFILES)),)
 # This allows to skip the maven jar build (for instance, if its builded by another jenkins job)
 # for instance with debuild -us -uc -b --build-profiles=nobuildjar
 # Also we can -DskipITs tests
-	mvn -Dmaven.repo.local=$(M2_REPO) clean package -DskipTests
+	mvn -Dmaven.repo.local=$(M2_REPO) clean package -P gbif-artifacts,livingatlas-artifacts,extra-artifacts -DskipTests
```

Comment:
I think `-P livingatlas-artifacts` will be enough

Reply:
Done. Thanks @muttcg.